dart/lib/src/protocol/sentry_event.dart

```diff
@@ -76,15 +78,20 @@ class SentryEvent {
 
   /// An object that was thrown.
   ///
-  /// It's `runtimeType` and `toString()` are logged. If this behavior is
-  /// undesirable, consider using a custom formatted [message] instead.
-  final dynamic exception;
+  /// It's `runtimeType` and `toString()` are logged.
+  /// If it's an Error, with a stackTrace, the stackTrace is logged.
+  /// If this behavior is undesirable, consider using a custom formatted [message] instead.
+  final dynamic throwable;
```

Comment:
`throwable` makes sense in Java, but not in Dart, maybe `error`? wdyt?

Reply:
IMO, as Dart uses both `Error`, and `Exception`, and we already have `SentryEvent.exception`, `error` would be more confusing, `throwable` seems a clearer way to indicate that it can be an `Error` or an `Exception`.

Reply:
The doc could be more explicite : `/// An Exception or an Error that was thrown.` ?

Reply:
what's about this then? https://github.com/getsentry/sentry-dart/pull/142
we have changed exactly for this reason

Reply:
well afaik it could be literally anything, it's a `dynamic` field, people can throw a String, so it should be generic enough to work for everything, and for that `throwable` makes sense.
maybe @bruno-garcia ?

Reply:
We made #142 before having `SentryException`, so it made sense to me. Now having an `error` field, which could be an `Error` or an `Exception`, and an `exception`, which is not an `Exception` but a `SentryException`, seems weird to me.

Reply:
✅ renamed "throwable" everywhere